obgyn/urls.py

```diff
@@ -42,6 +42,9 @@
     path('patient/', include('patient.urls')),
     path('opd/', include('patient_opd.urls')),
     path('report/', include('reports.urls')),
+    path('subscription/', include('subscription.urls')),
     path('template-header/', include('template_header.urls')),
+    path('subscription_purchase/', include('subscription_purchase.urls')),
```

Comment:
Check the patients.urls. 
include the subscription_purchase end point within subscription.urls. 
main urls.py should have minimum end points.